### PR TITLE
SAAS-7 Refactor dashboard tests

### PR DESCRIPTION
**Pull request by peterSirotnak**

### CODE REVIEW

File: ui-tests/components/instancesTable.ts [hunk 1]
```diff
@@ -1,32 +1,43 @@
...
-    removePmm: (serverName: string) => this.rowByText(serverName).getByTestId('pmm-instance-actions-remove'),
+    ...super.getTableElements(),
+    removePmm: (serverName: string) =>
+      this.elements.rowByText(serverName).getByTestId('pmm-instance-actions-remove'),
   };
 
-  fields = {};
+  fields = {
+    ...super.getTableFields(),
+  };
```
Comment (doracretu3pillar) [inline ui-tests/components/instancesTable.ts:17]: `fields` , `messages`  and `buttons` are never used. I recommend to add them when they are needed.

File: ui-tests/components/ticketsTable.ts [hunk 2]
```diff
@@ -18,6 +18,30 @@ export class TicketsTable extends Table {
...
+  };
+
+  buttons = {
+    ...super.getTableButtons(),
+  };
+
+  messages = {
+    ...super.getTableMessages(),
+  };
+
```
Comment (doracretu3pillar) [inline ui-tests/components/ticketsTable.ts:40]: same here for `fields`, `labels`, `buttons` , `messages`

File: ui-tests/components/instancesTable.ts (hunk 1 above)
Comment (peterSirotnak) [reply ui-tests/components/instancesTable.ts]: Fields removed

File: ui-tests/components/ticketsTable.ts (hunk 2 above)
Comment (peterSirotnak) [reply ui-tests/components/ticketsTable.ts]: Fields removed

**Review by doracretu3pillar — APPROVED** [review]

**Pull request merged**